runners/spark/src/main/java/org/apache/beam/runners/spark/translation/SparkGroupAlsoByWindowViaOutputBufferFn.java

```diff
@@ -52,7 +52,7 @@
  * An implementation of {@link GroupAlsoByWindowViaOutputBufferDoFn}
  * for the Spark runner.
  */
-public class SparkGroupAlsoByWindowFn<K, InputT, W extends BoundedWindow>
+public class SparkGroupAlsoByWindowViaOutputBufferFn<K, InputT, W extends BoundedWindow>
```

Comment:
Pretty sure this was in a separate commit, which is not optimal but still easier to review.

It's hard to avoid such changes while making big changes - not impossible, but hard - because my attention is dedicated to other "more important" things.
I could do this in a separate PR now, but it's already here... and simply a rename.
I tried to break-up as many things I could while working on this (Watermarks) but eventually the Runner API requires end-to-end work on many features just to pass the simplest `PAssert` so these things happen 😞